src/views/Editor/index.tsx

```diff
@@ -88,18 +92,46 @@ const Editor = (props: ViewProps): JSX.Element => {
 
   return (
     <>
-      <ViewHeader {...props} title="Editor" icon={PenBoxIcon} />
-
-      <main className="min-w-[30vw]">
-        <div className={`h-full relative ${!isSynced ? 'opacity-60' : ''}`}>
-          { /* @ts-expect-error yjsEditor needs more refinement */}
-          <TextbitEditable yjsEditor={editor} />
+      <Textbit>
+        <div className={`flex flex-col h-screen ${!isSynced ? 'opacity-60' : ''}`}>
+          <div className="grow-0">
+            <ViewHeader {...props} title="Editor" icon={PenBoxIcon} />
+          </div>
+
+          <ScrollArea>
+            <div className="overscroll-auto">
+              { /* @ts-expect-error yjsEditor needs more refinement */}
+              <TextbitEditable yjsEditor={editor} />
+            </div>
+          </ScrollArea>
+
+          <div className="grow-0 border-t opacity-90">
+            <Footer />
+          </div>
         </div>
-      </main>
+
```

Comment:
Maybe keep `<main />` here as well, or is that superfluous with `<Textbit />`?

Reply:
If we want to be picky we should probably not use `<main>` the way we do as we have several views open at the same time. The spec states:

*A document must not have more than one [main](https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element) element that does not have the [hidden](https://html.spec.whatwg.org/multipage/interaction.html#attr-hidden) attribute specified.*

And as the _hidden_ attribute tells the browser that the element should not be rendered we should not used that.

I think it is better we lean on having `<section>` in the View component.

Reply:
Yes, that sounds correct